components/BaseModal/BaseModal.tsx

```diff
@@ -93,6 +98,8 @@ const StyledCardBody = styled(Card.Body)`
 	overflow-y: scroll;
 `;
 
+const StyledResizableAndDraggableContainer = styled(Rnd)``;
```

Comment:
Since we don't use this for styling, I'm not sure we need to wrap the `Rnd` component with `styled`.

Reply:
It's just for sake of consistency on naming convention. Let me remove it.